app/views/shared/buttons/_ab_options_form.html.haml

```diff
@@ -9,7 +9,7 @@
       .col-md-8
         - if @edit
           = select_tag('button_type',
-                options_for_select(CustomButton::TYPES.invert, @edit[:new][:button_type]),
+                options_for_select(CustomButton::TYPES.invert.map{|request, index| [_(request), index]}, @edit[:new][:button_type]),
```

Comment:
This won't really work, since we don't collect those strings during string extraction.

Reply:
Sorry, I also don't really understand what you're saying here.

Reply:
So the `CustomButton::TYPES` constant comes from backend, right? And I don't see those two strings (the ones that you're applying gettext to here) being marked for translation in backend (i.e. you'd need to use `N_()` in backend to be able to use `_()` here).

Reply:
Not sure what's going on in the backend but I do know that the strings translate properly.

<img src="https://user-images.githubusercontent.com/64800041/97715809-32d3ec80-1a99-11eb-9014-12aa2b758534.png" width="500">

Reply:
Yes, but that's just a coincidence, that we already have those 2 strings in the catalog. For this fix to be complete, these two strings in `CustomButton::TYPES` constant need to be properly marked for translation.

Reply:
This should work right?
```
  TYPES = { "default"          => N_("Default"),
            "ansible_playbook" => N("Ansible Playbook")}.freeze
```

Reply:
made the PR here: https://github.com/ManageIQ/manageiq/pull/20762